packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-data-tile/OrganizationDataProtectionTile.component.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { DashboardTile, Links, LinkType } from '@ovhcloud/manager-components';
+import { OsdsChip } from '@ovhcloud/ods-components/react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { ODS_CHIP_SIZE } from '@ovhcloud/ods-components';
+import {
+  DATA_PROTECTION_BACKUP_TITLE,
+  DATA_PROTECTION_RECOVERY_TITLE,
+} from '@/pages/dashboard/organization/OrganizationDashboard.constants';
+import { useManagedVcdOrganizationBackup } from '@/data/hooks/useManagedVcdOrganization';
+import Loading from '@/components/loading/Loading.component';
+
+export default function DataProtectionTile() {
+  const { t } = useTranslation('dashboard');
+  const { id } = useParams();
+  const {
+    data: vcdOrganizationBackup,
+    isError,
+    isLoading,
+  } = useManagedVcdOrganizationBackup(id);
+
+  const backupStatus = vcdOrganizationBackup?.data?.resourceStatus;
+  const backupStatusMessage =
+    isError || !backupStatus
+      ? t('managed_vcd_dashboard_backup_status_error')
+      : t(`managed_vcd_dashboard_backup_status_${backupStatus}`);
+
+  return (
+    <div>
+      <DashboardTile
+        title={t('managed_vcd_dashboard_data_protection')}
+        items={[
+          {
+            id: 'backup',
+            label: DATA_PROTECTION_BACKUP_TITLE,
+            value: (
+              <div className="flex flex-col">
+                {/* TODO: check other ways to handle error (+refetch) inside a Tile
+                (<ErrorBanner /> is too big to be used here) */}
+                {isLoading && <Loading className="w-8" />}
+                {!isLoading && <span>{backupStatusMessage}</span>}
+                <Links
+                  type={LinkType.external}
+                  label={t('managed_vcd_dashboard_backup_link')}
+                  href={'/'} // TODO : replace with Veeam listing page link
```

Comment:
You will have to use shell.navigation for this

https://github.com/ovh/manager/pull/12611/files#diff-be7c2d2ab688d37454b09fad34ca61205590ea2f8f0ee298ee2f6c9fa4bb97b5

Example in OrganizationCell component